tempesta_fw/http.c

```diff
@@ -945,22 +961,13 @@ tfw_http_init(void)
 	return 0;
 }
 
-void
+static void
 tfw_http_exit(void)
 {
 }
 
-/**
- * Calculate key of a HTTP request by hashing its URI and Host header.
- *
- * Requests with the same URI and Host are mapped to the same key with
- * high probability. Different keys may be calculated for the same Host and URI
- * when they consist of many chunks.
- */
-unsigned long
-tfw_http_req_key_calc(const TfwHttpReq *req)
-{
-	return (tfw_hash_str(&req->h_tbl->tbl[TFW_HTTP_HDR_HOST].field) ^
-	        tfw_hash_str(&req->uri));
-}
-EXPORT_SYMBOL(tfw_http_req_key_calc);
+TfwCfgMod tfw_mod_http = {
+	.name = "http",
+	.init = tfw_http_init,
+	.exit = tfw_http_exit,
+};
```

Comment:
Why http.c is also treated as TfwCfgMod? It's not a good idea to declare TfwCfgMod for each C file.